napari_allencell_annotator/_tests/view/viewer_test.py

```diff
@@ -0,0 +1,113 @@
+from pathlib import Path
+
+import napari
+from bioio import BioImage
+
+import napari_allencell_annotator
+from napari.layers import Points
+
+from napari_allencell_annotator.util.image_utils import ImageUtils
+from napari_allencell_annotator.view.viewer import Viewer, PointsLayerMode
+import numpy as np
+import pytest
+from napari.utils import Colormap
+
+
+@pytest.fixture
+def viewer(make_napari_viewer: napari.Viewer) -> Viewer:
+    return Viewer(make_napari_viewer())
+
+
+def test_all_points_layer(viewer: Viewer) -> None:
+    # ARRANGE
+    viewer.viewer.add_shapes()
+    test_points_layer1: Points = viewer.create_points_layer("test1", "blue", True, 2)
+    test_points_layer2: Points = viewer.create_points_layer("test2", "blue", True, 2)
```

Comment:
We want to avoid mutating fixtures here: https://sapegin.me/blog/avoid-mutation/#:~:text=Mutation%20may%20lead%20to%20unexpected,vigilant%20when%20reading%20the%20code.